engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentStatusMapper.scala

```diff
@@ -31,9 +32,7 @@ class K8sDeploymentStatusMapper(definitionManager: ProcessStateDefinitionManager
     deployments match {
       case Nil => None
       case one :: Nil => Some(status(one, pods))
-      case duplicates =>
-        val errors = List(s"Expected one deployment, instead: ${duplicates.map(_.metadata.name).mkString(", ")}")
-        Some(definitionManager.processState(K8sStateStatus.MultipleJobsRunning, errors = errors))
+      case duplicates => Some(K8sProcessStateDefinitionManager.errorMultipleJobsRunning(duplicates))
```

Comment:
eg. on multiple you should be able to cancel scenario ;)

Reply:
Yes. Action cancel is allowed in `SimpleStateStatus.statusActionsPF`